pkg/elasticsearch/elasticsearch_service.go

```diff
@@ -23,3 +31,44 @@ func (es ElasticSearchService) GetStatus() int {
 
 	return service.ServiceStatusPending
 }
+
+func (es ElasticSearchService) GetActions() []action.IActionGroup {
+	return []action.IActionGroup{
+		action.ActionGroup{
+			Name: "Elasticsearch Action Group",
```

Comment:
Not a meaningful group name and two different kind of actions within the same group